check50.py

```diff
@@ -313,10 +315,18 @@ def import_checks(identifier):
     try:
         # Import module from file path directly.
         module = imp.load_source(slug, os.path.join(config.check_dir, "__init__.py"))
-        # Ensure that there is exactly one class decending from Checks defined in this package.
-        checks, = (cls for _, cls in inspect.getmembers(module, inspect.isclass)
-                   if hasattr(cls, "_Checks__sentinel")
-                   and cls.__module__.startswith(slug))
+        test_methods = [method for method in module.__dict__.values() if getattr(method, "is_check", False)]
```

Comment:
No reason to waste memory with a list comprehension here. Could use a generator comprehension but it's probably just simpler to replace this and the loop below with e.g.,

    for check in module.__dict__.values():
        if getattr(check, "is_check", False):
            setattr(CustomChecks, check.__name__, check)